pom.xml

```diff
@@ -30,7 +30,7 @@
         <maven.compiler.target>11</maven.compiler.target>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.reporting.outputEncoding>UTF-8</project.reporting.outputEncoding>
-        <vaadin.version>23.2-SNAPSHOT</vaadin.version>
```

Comment:
should we use 23.3-SNAPSHOT?

Reply:
True, changes. I'll merge if it passes! Thanks!